### ISSUE TITLE
Multiple connected panoramas generated?

### ISSUE THREAD

**Issue by antoinemacia** [ArcherFMY/SD-T2I-360PanoImage#2]
Hello, 

Wondering how multiple generated panoramas can be connected to one another (essentially maintaining the style as well as whatever artefacts/furnitures have been generated)

Use case: multiple panoramas for a home

**Comment by ArcherFMY** [comment]
Hi,

It is a very interesting application and we have discussed within the team. 

We think one straightforward method is that we can generate a main pano-image (like a living room) and convert it to a 3D structure using depth. Then find a new location to render a sparse pano-image and fill the holes via inpainting. 

Another data-driven idea is that we can treat it as a pano-video generation problem. The AnimateDiff (https://animatediff.github.io/) can be a try. The motion module may help to maintain the continuity of different panoramas. But we have not tried yet.

**Comment by antoinemacia** [comment]
Yeah nice! 

The first option is pretty sound, [this project]( https://garrickbrazil.com/omni3d/) could be of interest (I believe does more or less what you're describing)

Im curious how styled could be maintained across rooms however...

**Comment by ArcherFMY** [comment]
> Yeah nice!
> 
> The first option is pretty sound, [this project](https://garrickbrazil.com/omni3d/) could be of interest (I believe does more or less what you're describing)
> 
> Im curious how styled could be maintained across rooms however...

Maybe we can train a LoRA for a certain decoration style :-)

**Comment by antoinemacia** [comment]
Yeah nice! Also realise I have shared the wrong paper previously in regards to depth conversion, this is what I meant to share https://cyhsu14.github.io/OmniNeRF/

**Comment by ArcherFMY** [comment]
> Yeah nice! Also realise I have shared the wrong paper previously in regards to depth conversion, this is what I meant to share https://cyhsu14.github.io/OmniNeRF/

Interesting! Thanks for sharing!

**Comment by zhanghongyong123456** [comment]
> Yeah nice! Also realise I have shared the wrong paper previously in regards to depth conversion, this is what I meant to share https://cyhsu14.github.io/OmniNeRF/

Do you have any implementation cases? I found that a video is the effect you want, but I don't know what the implementation process is.

https://github.com/ArcherFMY/SD-T2I-360PanoImage/assets/48466610/e5745e49-da72-4017-92ec-c78face9c6b5
